crypto/ecdsa/ecdsa.go

```diff
@@ -0,0 +1,38 @@
+package ecdsa
+
+import (
+	"crypto/ecdsa"
+
+	"github.com/Layr-Labs/eigensdk-go/types"
+	"github.com/ethereum/go-ethereum/crypto"
+)
+
+// SignMsg is a wrapper around geth's crypto.Sign function which adds 27 to the v value
+// see https://github.com/ethereum/go-ethereum/issues/28757#issuecomment-1874525854
+// and https://twitter.com/pcaversaccio/status/1671488928262529031
+// Currently all of our contracts expect the v value to be 27 or 28, which is why we add it here.
+// VerifySignature below thus subtracts 27 from the v value before verifying the signature.
+// TODO(samlaf): if and when we figure out some avs contracts use the 0/1 v scheme, or the eip-155 scheme,
+// then we should generalize this function to sign based on a scheme passed as argument.
+func SignMsg(msg []byte, privateKey *ecdsa.PrivateKey) ([]byte, error) {
```

Comment:
This is an issue that I'm unsure how to deal with, so arranged it like this for now, but if anyone has ideas on how to better deal with this, would be good to know.